boilerplate/app/app.tsx

```diff
@@ -11,14 +11,14 @@
  */
```

Comment:
Changes to this file include removing the Storybook wrapper as well as adding font-loading (the old way was a placeholder anyways).